extended/src/main/java/io/kubernetes/client/extended/leaderelection/resourcelock/ConfigMapLock.java

```diff
@@ -93,8 +94,8 @@ public boolean create(LeaderElectionRecord record) {
           coreV1Client.createNamespacedConfigMap(namespace, configMap, null, null, null);
       configMapRefer.set(createdConfigMap);
       return true;
-    } catch (Throwable t) {
-      log.error("failed to create leader election record as {}", t.getMessage());
+    } catch (ApiException e) {
+      log.error("received {} when creating configmap lock", e.getCode(), e);
```

Comment:
I think we expect the possibility of `409` here too, so we should do the same `info/error` check here.

multiple different containers all starting at the same time will all initially try to create the lock, see the code here:

https://github.com/kubernetes-client/java/blob/master/extended/src/main/java/io/kubernetes/client/extended/leaderelection/LeaderElector.java#L248

cc @yue9944882 to double-check my understanding.

Reply:
i think so. an `ApiException` will also be thrown here upon `409` but it's actually an expected behavior so it will be better if it's info-level instead of alerting error messages

Reply:
ah, yes, you are right! fixed